bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
+
+#----install pigpio (npm install pigpio will be exec by tjbotlib package.json)
+curl abyz.co.uk/rpi/pigpio/pigpio.zip > pigpio.zip
```

Comment:
oh right right right, I've seen this too -- need to `sudo apt-get install pigpio` before `npm install pigpio` will work, because otherwise it `node-gyp` fail while compiling the native module when the pigpio headers are missing

i recommend just adding `pigpio` to the `sudo apt-get install -y` line above (line 36?)